packages/design-system-react-native/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/design-system-react-native",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Bumping `@metamask/design-system-react-native` minor version as we are adding functionality without any breaking changes